tutorials/understanding-group/index.md

```diff
@@ -250,9 +266,11 @@ The following is the content of the `policy.json`. It states that:
 Have the group administrator create the group policy with metadata that identifies it as one with a quick turnaround:
 
 ```sh
-$ simd tx group create-group-policy $ALICE $GROUP_ID "quick turnaround" policy.json
+$ simd tx group create-group-policy $ALICE $GROUP_ID "{\"name\":\"quick turnaround\",\"description\":\"\"}" policy.json
 ```
 
+Note that this time, the metadata is on-chain an escaped JSON string. This is as per the [group documentation](https://docs.cosmos.network/v0.46/modules/group/06_metadata.html).
```

Comment:
My understanding is that the _escaping_ is for the command line only. After it has been saved, there are no more `\`.

Reply:
No, it will stay escaped, as the `Metadata` field is a string: https://github.com/cosmos/cosmos-sdk/blob/5deb137f7a83506381472928d898cd823735aade/x/group/tx.pb.go#L73